src/positioner/src/Positioner.js

```diff
@@ -119,6 +119,13 @@ export default class Positioner extends PureComponent {
     this.state = initialState()
   }
 
+  componentDidMount() {
+    // If `isShown` is `true` on first render, <Transition> won't emit any enter/entered events
+    // because there was no transition from `false` to `true`
+    // In order to still display and position the content properly, `update()` must be fired after first render
+    this.update()
```

Comment:
Should transition emit on appear instead of this?

Reply:
That's a good idea, will try that.